### ISSUE TITLE
Answer bugs

### ISSUE THREAD

**Issue by oguzkurt** [XimeraProject/ximeraLatex#52]
This is a specific instance of `$\answer{..}$` environment. 

I am using it outside any specific ximera environment to teach syntax so that they will not try to calculate things like `\binom{100}{20}` on a calculator. I needed to use multiple answer boxes. When I published it,
the first answer box worked wonderful and the other boxes would turn green as soon as I clicked within the box  without me even writing anything or pushing `ENTER` or entering the right answer.

Any idea as to how to fix this bug?

On the same page, I have another one within example environment and it works fine!

**Comment by bartsnapp** [comment]
Hi,

All answers need to be within a "thm" like environment.

Put it within "exercise" and it should work


On Feb 14, 2017 11:57 AM, "oguzkurt" <notifications@github.com> wrote:

This is a specific instance of <$\answer{..}$> environment.

I am using it outside any specific ximera environment to teach syntax so
that they will not try to calculate things like <\binom{100}{20}> on a
calculator. I needed to use multiple answer boxes. When I published it,
the first answer box worked wonderful and the other boxes would turn green
as soon as I clicked within the box without me even writing anything or
pushing "" or entering the right answer.

Any idea as to how to fix this bug?

On the same page, I have another one within example environment and it
works fine!

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/XimeraProject/ximeraLatex/issues/52>, or mute the thread
